module/localisation/BallLocalisation/data/config/BallLocalisation.yaml

```diff
@@ -23,3 +23,18 @@ acceptance_radius: 2
 
 # Maximum number of ball rejections before acceptance
 max_rejections: 25
+
+# Robot to robot communication parameters
+use_r2r_balls: true
+
+# Number of robots on our team
+max_robots: 3
```

Comment:
I'd probably do this dynamically. If the vector isn't of size player ID, then resize it. Then we don't need to encode it specifically.